docs/topic_guides/settings.md

```diff
@@ -0,0 +1,534 @@
+2. What **pairs of records** to consider (defined by [blocking rules](./blocking_rules.md))
+3. What **features** to consider, and how they should be **compared** (defined by [comparisons](./customising_comparisons.ipynb))
+
+
+## Defining a Splink model with a settings dictionary
+
+All aspects of a Splink model are defined via the settings dictionary. The settings object is a json-like object which underpins a model.
+
+For example, consider a simple model (as defined in the [README](.../README.md)). The model is defined by the following settings dictionary
+
+```py linenums="1"
+import splink.duckdb.duckdb_comparison_library as cl
+import splink.duckdb.duckdb_comparison_template_library as ctl
+
+settings = {
+    "link_type": "dedupe_only",
+    "blocking_rules_to_generate_predictions": [
+        "l.first_name = r.first_name",
+        "l.surname = r.surname",
+    ],
+    "comparisons": [
+        ctl.name_comparison("first_name"),
+        ctl.name_comparison("surname"),
+        ctl.date_comparison("dob", cast_strings_to_date=True),
+        cl.exact_match("city", term_frequency_adjustments=True),
+        ctl.email_comparison("email"),
+    ],
+}
+```
+
+Where:
+
+**1. Type of linkage**
+
+The `"link_type"` is defined as a deduplication for a single dataset.
+
+```py linenums="5"
+    "link_type": "dedupe_only", 
+```
+
+**2. Pairs of records to consider**
+
+The `"blocking_rules_to_generate_predictions"` define a subset of pairs of records for the model to be trained on where there is a match on `"first_name"` or `"surname"`.
+
+```py linenums="6"
+    "blocking_rules_to_generate_predictions": [
+        "l.first_name = r.first_name",
+        "l.surname = r.surname",
+    ],
+```
+
+**3. Features to consider, and how they should be compared**
+
+The `"comparisons"` define the features to be compared between records: `"first_name"`, `"surname"`, `"dob"`, `"city"` and `"email"`.
+
+```py linenums="10"
+    "comparisons": [
+        ctl.name_comparison("first_name"),
+        ctl.name_comparison("surname"),
+        ctl.date_comparison("dob", cast_strings_to_date=True),
+        cl.exact_match("city", term_frequency_adjustments=True),
+        ctl.email_comparison("email"),
+    ],
+```
+
+Using functions from the [comparison template library](comparison_templates.ipynb) and [comparison library](./customising_comparisons.ipynb#method-1-using-the-comparisonlibrary) to define **how** these features should be compared.
+
+```py linenums="1"
+import splink.duckdb.duckdb_comparison_library as cl
+import splink.duckdb.duckdb_comparison_template_library as ctl
+``` 
+
+For more information on comparisons are defined, see the [dedicated topic guide](./customising_comparisons.ipynb).
+
+These functions generate comparisons within the settings dictionary. See below for the full settings dictionary once the `comparison_library` and `comparison_template_library` functions have been evaluated:
```

Comment:
You can probably ignore this one, but this line is a little ambiguous for a new user. I don't think my suggestion helps that much either though.

```suggestion
These functions generate comparisons within the settings dictionary. See below for the full settings dictionary once the `comparison_library` and `comparison_template_library` functions have been evaluated and constructed:
```